script/Airdrop.s.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import { Script } from "forge-std/Script.sol";
+import { console } from "forge-std/console.sol";
+
+import { Airdrop } from "../src/distribution/Airdrop.sol";
+
+// Command to deploy the airdrop contract:
+// forge script script/Airdrop.s.sol --rpc-url $ETH_RPC_URL --sender $DEPLOY_ADDRESS --keystore $KEYSTORE_PATH --broadcast --verify -vvvv
```

Comment:
This comment should seemingly be moved to the README, with some background on environment setup.  For example `--verify` requires defining `$ETHERSCAN_API_KEY` in environment.  That should work fine on BaseSepolia, but unsure how contract verification works on our appchain.

Reply:
README should also mention that the EOA deploying the contract will become the initial owner.

Reply:
Nice; if no other setup is needed, I'm happy leaving it as-is.